.travis.yml

```diff
@@ -3,8 +3,6 @@ before_script:
   - git config --global user.name "Your Name"
 script: bundle exec rspec
 rvm:
-  - 1.9.3
-  - 2.0.0
-  - 2.1.0
-  - rbx
+  - 2.2.3
+  - 2.4.0
```

Comment:
@htanata The Ruby versions we specify in our build config provide the strongest indication of what version we support. When we change our Ruby version support, would that qualify as a **major** (not minor/patch) version change for the gem? What are your thoughts?